src/TodoListItem.js

```diff
@@ -1,5 +1,11 @@
 import React from "react";
 
-const TodoListItem = ({ item }) => <li>{item.title}</li>;
+const TodoListItem = ({ item, onRemoveClicked }) =>
+    <li>
+        <div style={{ display: "flex", margin: "4px 20px 4px 0", justifyContent: "space-between" }}>
+            {item.title || "\u00a0"}
```

Comment:
Nitpick (doesn't require changes): 
Instead of displaying a non-breaking space if a title isn't present, I'd recommend preventing an empty title from being submitted. This is a more common practice to validate the input on submit to determine if the form is valid. If invalid, prevent the form submit action from completing! Here's a simple example of how to prevent an empty title within `src/AddTodoForm.js`:

```js
...
    const handleAddTodo = (event) => {
        event.preventDefault();

        // this may look a bit strange, but it's an inverse of a 0 (invalid) or 1+ character (valid) boolean check 
        // or in simple terms: "run this code if todoTitle doesn't have a length"
        // for example by default: 
        // 0 is false and 1+ is true, 
        // but we want the opposite:
        // not 0 is false (valid field; bypass alert)
        // not 1+ is true (invalid field; show alert)
        // alternatively, you could just write:
        // todoTitle.length === 0, but the code below is subjectively cleaner
        if (!todoTitle.length) {
           alert("You must provide a todo title!");
           return; // early return to prevent executing the code below
        }

        onAddTodo({
            title: todoTitle,
            id: Date.now()
        });
        setTodoTitle(INITIAL_TITLE_VALUE);
    };
...
```

Reply:
Thanks for the idea! That's a much better solution. I added it in and also trimmed the whitespace before checking the length. For some reason if the title is just empty whitespace, it doesn't get added to the page, so the button gets moved all the way to the left. This eliminates that issue though. Thanks again!